### PR TITLE
Commit id plugin

### PR DESCRIPTION
**Pull request by pared**
Added null injection

### CODE REVIEW

File: spring-boot/src/test/java/com/baeldung/git/CommitIdTest.java [hunk 1]
```diff
@@ -11,27 +11,24 @@
...
-    @Value("${git.commit.message.short}")
+    @Value("${git.commit.message.short:#{null}}")
     private String commitMessage;
 
-    @Value("${git.branch}")
+    @Value("${git.branch:#{null}}")
     private String branch;
 
-    @Value("${git.commit.id}")
+    @Value("${git.commit.iddd:#{null}}")
```
Comment (pivovarit) [inline spring-boot/src/test/java/com/baeldung/git/CommitIdTest.java]: iddd?

File: testgitrepo [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit b3e635b96d114e25ca1da696acfd881b5fac28a7
```
Comment (pivovarit) [inline testgitrepo]: Unnecessary

File: spring-boot/src/test/java/com/baeldung/git/CommitIdTest.java (hunk 1 above)
Comment (pared) [inline spring-boot/src/test/java/com/baeldung/git/CommitIdTest.java]: done

File: testgitrepo (hunk 2 above)
Comment (pared) [inline testgitrepo]: done

**Pull request merged**